_data/ArpitaGanatra/ArpitaGanatra.md

```diff
@@ -0,0 +1,7 @@
+---
+name: ARPITA GANATRA # No longer than 28 characters
+institution: GANDHINAGAR INSTUTUTE OF TECHNOLOGY 🚩 # no longer than 58 characters
+profile_pic: ArpitaGanatra.png # Name and extension of your profile picture(ex. mona.png) The picture must be squared and 544px on width and height.
```

Comment:
Your profile picture extension does not match with the file (image is a ```.jpg```, not a ```.png```).